Nasal/c172p.nas

```diff
@@ -393,6 +393,17 @@ var reset_system = func {
     props.globals.getNode("/fdm/jsbsim/pontoon-damage/right-pontoon", 0).setIntValue(0);
 
     setprop("/engines/active-engine/kill-engine", 0);
+
+    if (getprop("/fdm/jsbsim/position/persistent")) {
+        setprop("position/latitude-deg", getprop("/sim/model/c172p/currentlat"));
```

Comment:
You have a lot of properties in the same directory that are related to each other (`/sim/model/c172p/current*`). Maybe you could group them together and move them to a separate directory like `/sim/model/c172p/current-position/*`)?